src/server/game/Conditions/ConditionMgr.h

```diff
@@ -132,9 +132,9 @@ enum ConditionSourceType
     CONDITION_SOURCE_TYPE_SMART_EVENT                    = 22,
     CONDITION_SOURCE_TYPE_NPC_VENDOR                     = 23,
     CONDITION_SOURCE_TYPE_SPELL_PROC                     = 24,
-    CONDITION_SOURCE_TYPE_TERRAIN_SWAP                   = 25, // don't use on 3.3.5a
-    CONDITION_SOURCE_TYPE_PHASE                          = 26, // don't use on 3.3.5a
-    CONDITION_SOURCE_TYPE_GRAVEYARD                      = 27, // don't use on 3.3.5a
+    CONDITION_SOURCE_TYPE_UNUSED_25                      = 25, // placeholder
+    CONDITION_SOURCE_TYPE_UNUSED_26                      = 26, // placeholder
+    CONDITION_SOURCE_TYPE_UNUSED_27                      = 27, // placeholder
```

Comment:
why? I think it might be useful to know that these conditions do exist but they are not used in 3.3.5a

Reply:
But why would you keep conditions that do not work for 3.3.5?

Reply:
I'm not saying to keep the condition, just the information that's telling you why that condition is not there

Reply:
So an additional comment?

Reply:
yes

Reply:
but also preserving the original name, why not

Reply:
I'd propose this in that case, so ppl know that they still can use this if they create a new condition
```cpp
CONDITION_SOURCE_TYPE_UNUSED_27                      = 27, // placeholder (CONDITION_SOURCE_TYPE_GRAVEYARD not used in 3.3.5)

Reply:
> why? I think it might be useful to know that these conditions do exist but they are not used in 3.3.5a

?? they are not used in 3.3.5a? because you think conditions = blizzard system?

Reply:
no

Reply:
> I'd propose this in that case, so ppl know that they still can use this if they create a new condition
> 
> ```c++
> CONDITION_SOURCE_TYPE_UNUSED_27                      = 27, // placeholder (CONDITION_SOURCE_TYPE_GRAVEYARD not used in 3.3.5)
> ```

this :)

Reply:
Sure I can add that